pkg/hintrunner/zero/zerohint_dictionaries.go

```diff
@@ -10,6 +10,60 @@ import (
 	f "github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
 )
 
+// DictNew hint creates a new dictionary
+//
+// `newDictNewHint` takes no operander as argument
+func newDictNewHint() hinter.Hinter {
```

Comment:
Let's explain a little more. What's the difference between DictNew and DefaultDictNew